cmd/zed/dev/compile/command.go

```diff
@@ -47,6 +48,10 @@ Zed kernel operators.
 	New: New,
 }
 
+func init() {
+	dev.Cmd.Add(Cmd)
+}
+
```

Comment:
I can't leave a comment at the site, but `nodeProblem` down below has a "zed compile" that should be changed to "zed dev compile".